.github/workflows/wpt-chromedriver.yml

```diff
@@ -10,6 +10,7 @@ on:
 
 jobs:
   wpt:
+    timeout-minutes: 20
```

Comment:
I would put it on the step rather then the job, we have observe some slow set-ups from time to time.

Reply:
Didn't know it was possible. Done!